qemu/tests/qemu_guest_agent.py

```diff
@@ -894,6 +894,38 @@ def gagent_check_thaw_unfrozen(self, test, params, env):
             test.fail("The return value of thawing an unfrozen fs is %s,"
                       "it should be zero" % ret)
 
+    @error_context.context_aware
+    def gagent_check_query_chardev(self, test, params, env):
+        """
+        Check guest agent service status through QMP 'query-chardev'
+
+        :param test: kvm test object
+        :param params: Dictionary with the test parameters
+        :param env: Dictionary with test environment
+        """
+        def check_value_frontend_open(out, expected):
+            """
+            Get value of 'frontend-open' after executing 'query-chardev'
+
+            :param out: output of executing 'query-chardev'
+            :param expected: expected value of 'frontend-open'
+            """
+            for i in out:
+                if "org.qemu.guest_agent.0" in str(i):
+                    ret = i["frontend-open"]
+                    if str(ret) == expected:
```

Comment:
If the `ret` is a Boolean type. I'd suggest define `expected` as a Boolean as well.
So you can put it this way`if ret is expected:`, instead of converting it to a str